Makefile

```diff
@@ -35,11 +39,17 @@ ${RUSTC_BOEHM_DIR}:
 	git clone https://github.com/softdevteam/rustc_boehm
 
 ${YKSOM_BASELINE}:
-	cd ${YKSOM_DIR} && RUSTFLAGS="-l kruntime -L `realpath ../krun/libkrun`" cargo +nightly build --release --features "krun_harness" --target-dir=${YKSOM_BASELINE}
+	cd ${YKSOM_DIR} && RUSTFLAGS="-l kruntime -L ${LIBKRUNPATH}" cargo +nightly build --release --features "krun_harness" --target-dir=${YKSOM_BASELINE}
 
 ${YKSOM_CUSTOM_RUSTC}:
 	cd rustc_boehm && ./x.py build --stage 1 && rustup toolchain link rustc_boehm build/x86_64-unknown-linux-gnu/stage1
-	cd ${YKSOM_DIR} && RUSTFLAGS="-l kruntime -L `realpath ../krun/libkrun`" cargo +rustc_boehm build --release --features "rustc_boehm krun_harness" --target-dir=${YKSOM_CUSTOM_RUSTC}
+	cd ${YKSOM_DIR} && RUSTFLAGS="-l kruntime -L ${LIBKRUNPATH}" cargo +rustc_boehm build --release --features "rustc_boehm krun_harness" --target-dir=${YKSOM_CUSTOM_RUSTC}
+
+${SOMPP_DIR}:
+	git clone --recursive https://github.com/softdevteam/SOMpp && \
+	cd ${SOMPP_DIR} && mkdir build && \
```

Comment:
Since these lines are continuations, we probably want to indent them a couple of spaces.

Reply:
Ok I'll try this. I'm always scared to do whitespace formatting in Makefiles as I never remember the rules.

Reply:
Fixed: 2aafb3344616bcac6e746937e9b7acdcdc3b1385